src/main/java/in/org/projecteka/hiu/SecurityConfiguration.java

```diff
@@ -53,8 +54,12 @@
 
     private static final List<Map.Entry<HttpMethod, String>> CM_PATIENT_APIS = List.of(
             Map.entry(HttpMethod.GET, "/cm/hello"),
-            Map.entry(HttpMethod.POST, APP_PATH_PATIENT_CONSENT_REQUEST));
+            Map.entry(HttpMethod.POST, APP_PATH_PATIENT_CONSENT_REQUEST),
+            Map.entry(HttpMethod.POST, HealthInfoController.API_PATH_FETCH_PATIENT_HEALTH_INFO));
 
+    private static final List<Map.Entry<HttpMethod, String>> DOCTOR_AND_PATIENT_COMMON_APIS = List.of(
+            Map.entry(HttpMethod.GET, HealthInfoController.API_PATH_GET_ATTACHMENT)
+    );
```

Comment:
Did you change the HIU-UI to pass the token?

Reply:
Not yet. Need to change it.